src/DGtal/geometry/curves/estimation/LambdaMST3D.h

```diff
@@ -100,27 +103,40 @@ namespace DGtal {
      * Checks the validity/consistency of the object.
      * @return 'true' if the object is valid, 'false' otherwise.
      */
-    bool isValid() const;
+    bool isValid ( ) const;
     
     /**
-     * @param it ConstIterator defined over the underlying curve
+     * For ranges of points the second version of this method is faster
+     * than iterating over this version.
+     *
+     * NOTE THAT ONLY THIS VERSION ALLOWS FOR DSS FILTRATION!
+     *
+     * @param p a point of the underlying curve
      * @return tangent direction
      */
-    RealVector eval ( const ConstIterator & it );
+    RealVector eval ( const Point & p );
     
     /**
      * @tparam OutputIterator writable iterator.
      * More efficient way to compute tangent directions for all points of a curve.
+     *
      * @param itb begin iterator
      * @param ite end iterator
      * @param result writable iterator over a container which stores estimated tangent directions.
      */
     template <typename OutputIterator>
-    OutputIterator eval ( const ConstIterator & itb, const ConstIterator & ite, 
-                        OutputIterator result );
+    OutputIterator eval ( ConstIterator itb, ConstIterator ite, OutputIterator result );
+
+    /**
+     *
+     * @return the internal dss filter
+     */
+    DSSFilter & getDSSFilter ( );
     
     // ------------------------- Internals ------------------------------------
   protected:
+
+      typedef typename std::vector<SegmentComputer >::const_iterator OrphantDSSIterator;
```

Comment:
It should be orphan. It refers to points that are not covered anymore because the corresponding DSSes were filtered out, e.g., these DSSes were too short.